example/install_rackhd.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+#
+########################################################################################################
+#
+#  this script is based on RackHD document "Ubuntu Package Based Installation"
+#
+#  http://rackhd.readthedocs.io/en/latest/rackhd2.0/ubuntu_package_installation.html?highlight=install
```

Comment:
Need to update this link, I recommend:

`http://rackhd.readthedocs.io/en/latest/rackhd/ubuntu_package_installation.html`